src/rpc/handlers/NFTOffers.cpp

```diff
@@ -126,6 +123,7 @@ enumerateNFTOffers(
     for (auto const& offer : offers)
         appendNftOfferJson(offer, jsonOffers);
 
+    response.insert_or_assign(JS(offers), std::move(jsonOffers));
```

Comment:
Why do we need this syntax versus `response[JS(offers)] = std::move(jsonOffers);`

Reply:
operator[] returns a reference to the element inside of the response whereas `insert_or_assign` accepts an rvalue-ref. this means that `response[...] = std::move(...)` will default-construct and then move-assign. the `insert_or_assign` version will just call the move constructor.